python/ray/rllib/agents/dqn/simple_q_model.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from ray.rllib.models.tf.tf_modelv2 import TFModelV2
+from ray.rllib.utils import try_import_tf
+
+tf = try_import_tf()
+
+
+class SimpleQModel(TFModelV2):
```

Comment:
@gehring , I've tried pulling out a minimal DQN and Q model. Any further thoughts on how to improve this?

There is a bit of magic btw in the model catalog, which mixes in this default implementation into other network model (e.g., fcnet, convnet), so that you don't *have* to implement get_action_scores if you don't want to -- only if you want to customize it.